srcpkgs/anydesk/template

```diff
@@ -1,31 +1,36 @@
 # Template file for 'anydesk'
 pkgname=anydesk
-version=6.2.1
+version=6.3.0
 revision=1
-archs="x86_64"
+archs="x86_64 armv7l armv7hf"
 create_wrksrc=yes
 hostmakedepends="patchelf"
 depends="hicolor-icon-theme gtkglext"
 short_desc="Fast remote desktop application"
 maintainer="mobinmob <mobinmob@disroot.org>"
 license="custom:Proprietary"
-#Free for personal use. Terms and conditions:https://anydesk.com/en/terms
+# Free for personal use. Terms and conditions: https://anydesk.com/en/terms
 homepage="https://anydesk.com/remote-desktop"
 nopie=yes
 restricted=yes
 
-distfiles="https://download.anydesk.com/linux/anydesk_${version}-1_x86_64.rpm
-	https://download.anydesk.com/linux/anydesk-${version}-amd64.tar.gz"
-checksum="68f8395a0c84f2560ffa425497fbf9846a61bb2dbc4b1010a8ffca7856514ab2
- 96a7ded2144ecdaff381c34e49eee3275caaaac00847e91abf5e7783706c989c"
-
+case "${XBPS_TARGET_MACHINE}" in
+	x86_64)
+		distfiles="https://download.anydesk.com/linux/anydesk_${version}-1_amd64.deb"
+		checksum="30f6e44cea28428d1cb645055fae72cf166f63318e0c94f2c12d18a3614ce30c"
+		;;
+	armv7*)
+		distfiles="https://download.anydesk.com/rpi/anydesk_${version}-1_armhf.deb"
+		checksum="10a8f442877d65458c800da2a4c772532c178a675c55604cca66156a5c812e1a"
+		;;
+esac
 
 do_build() {
 	patchelf --remove-needed libpangox-1.0.so.0 usr/bin/anydesk
 }
 
 do_install() {
-	vlicense "${pkgname}-${version}"/copyright
-	rm -rf "${pkgname}-${version}"
-	vcopy "*" /
+	echo "See: https://anydesk.com/en/terms" > LICENSE
```

Comment:
This is not an acceptable license. Fetch the web content at build time place that content in the LICENSE file.

Reply:
with T&C like (it's not just a copyright licence) this the ones that apply are usually the ones that exist at the present moment, not what existed at build time, so for packages like this and chrome and discord I've moved them to this style.

Reply:
You agree to the terms when you install the software. Unless those terms expressly include a clause that you agree to abide by any future changes to the terms, the provider is not able to change the terms after the fact. Regardless, it would be most appropriate to capture the terms as they exist when they are enacted.

If there's an argument that build and install time might be materially different (which is less likely to be true for restricted packages), the right thing to do is fetch the license in an INSTALL script.

Reply:
The license is included in the generic linux build, which is x64 only. I just opted for a cleaner resolution (suggested by @classabbyamp) in which both of the builds are fetched from the deb packages, which do not include the license file.

As it's a service based software, I presume the license and T&C might change over time, so in order to be up to date with the license/T&C, it's best to refer the user to the web page, instead of looking at an older license file which is not up to date.

The software is backwards compatible (can work/connect) with older versions of it, up to 2 or 3 major versions behind the current build, so it is actually more practical to include the current "state of things" rather than what used to be "the norm" at install time.

The Windows installer has no T&C/license acceptance when installing (maybe it has, but I haven't noticed to be honest). The Debian builds as well (they don't even have the license file in the packages, only the generic Linux builds have that), so I really can't see a problem if the user is just pointed at a web page to look at the current state of legal affairs regarding using the software.